.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: c
+sudo: required
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+    - os: linux
+      dist: precise
+      sudo: required
+install:
+- sudo apt-get update
+- sudo apt-get install --yes build-essential autoconf libtool gawk alien fakeroot gdebi linux-headers-generic
+- sudo apt-get install zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev parted lsscsi wget ksh
+script:
+- export MAKEFLAGS="-j1"; echo $MAKEFLAGS
```

Comment:
Actually, this should be dropped entirely since it shouldn't speed anything up.  When the packages are built the correct `-jN` should already be used.  This is done in the RPM spec file with the `%{?_smp_mflags}` macro.

```
%build
%configure \
    --with-config=user \
    --with-udevdir=%{_udevdir} \
    --with-udevruledir=%{_udevruledir} \
    --with-dracutdir=%{_dracutdir} \
    --disable-static \
    %{debug} \
    %{systemd}
make %{?_smp_mflags}
```